test/calculus_tests.jl

```diff
@@ -39,9 +39,11 @@ function runtests()
 				nrank_per_block = 0 # dummy value
 				irank = 0 # dummy value
 				comm = MPI.COMM_NULL # dummy value 
+                element_spacing_option = "uniform" # dummy value
```

Comment:
Would be good to test "sqrt" spacing. I think it should pass this test? Instead of setting `element_spacing_option` here, replace line 17 above (Github won't let me put a comment that far away from changes) with
```julia
(discretization, element_spacing_option) ∈ (("finite_difference", "uniform"), ("chebyshev_pseudospectral", "uniform"), ("chebyshev_pseudospectral", "sqrt")),

Reply:
Making this change leads to tests which fail. I will look in to this as soon as I have bandwidth!